.github/travis/script.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
```

Comment:
Looks like this file isn't needed anymore, since all its actions have been split up in parallel?

Reply:
Done.